Documentation/Books/AQL/Examples/Join.md

```diff
@@ -220,6 +220,16 @@ So, for each user we pick the list of their friends and count them. The ones whe
 count equals zero are the lonely people. Using *RETURN 1* in the subquery
 saves even more precious CPU cycles and gives the optimizer more alternatives.
 
+### Index usage
+
+Especially on joins you should [make sure indices can be used to speed up your query.](../../ExecutionAndPerformance/ExplainingQueries.md)
```

Comment:
The link caused an error in my local build. I had to correct it to `../ExecutionAndPerformance/ExplainingQueries.md`. Wasn't this caught by the docu build script / CI?